src/scancode/agent/scancode_wrapper.cc

```diff
@@ -62,42 +62,28 @@ unsigned getFilePointer(const string &filename, size_t start_line,
  *
  * @see https://scancode-toolkit.readthedocs.io/en/latest/cli-reference/list-options.html#all-basic-scan-options
  */
-string scanFileWithScancode(const State &state, const fo::File &file) {
-
-  FILE *in;
-  char buffer[512];
+void scanFileWithScancode(const State &state, string file_location, string outputFile) {
   string projectUser = fo_config_get(sysconfig, "DIRECTORIES", "PROJECTUSER",
-    NULL);
+  NULL);
   string cacheDir = fo_config_get(sysconfig, "DIRECTORIES", "CACHEDIR",
-    NULL);
+  NULL);
 
   string command =
-      "PYTHONPATH='/home/" + projectUser + "/pythondeps/' " +
-      "SCANCODE_CACHE=" + cacheDir + "/scancode " + // Use fossology's cache
-      "/home/" + projectUser + "/pythondeps/bin/scancode -" +
-      state.getCliOptions() +
-      " --custom-output - --custom-template scancode_template.html " +
-      file.getFileName() + " --quiet " +
-      ((state.getCliOptions().find('l') != string::npos) ? " --license-text --license-score " +
-      to_string(MINSCORE): "");
```

Comment:
Please check if the API or `runscanonfile.py` script can make use of this `MINSCORE` which defines a minimum threshold of scancode confidence above which only the result will be stored in DB.

Reply:
Yes, we can also paas `MINSCORE` to API.

Reply:
Will make changes accordingly.